### ISSUE TITLE
Format colors

### ISSUE THREAD

**Issue by zomer-g** [OriHoch/data4dappl#79]
PDF no change (#e0051e)
TIFF should be #e0051e instead of #AAAAAA

XLX and XLSX BACKGROUND should be #237B48 instead of #32B267
XSL BACKGROUND should be #237B48 instead of #AAAAAA

ICS BACKGROUND should be #796901 instead of #aaaaaa

CSV BACKGROUND should be #856A00 instead of #DFB100

XML BACKGROUND should be #AD5100 instead of #856A00

QGIS BACKGROUND should be #AE16D4 instead of #aaaaaa

ZIP BACKGROUND should be #6B6B6B instead of #777777

DOC, DOCX BACKGROUND should be #5254FF instead of #aaaaaa

PPT, PPTX BACKGROUND should be #9A5B0E instead of #aaaaaa

any other BACKGROUND should be #000000 instead of #777777

**Comment by OriHoch** [comment]
done + deployed
